package.json

```diff
@@ -4,11 +4,26 @@
   "description": "",
   "main": "cypress.config.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "cy:open": "cypress open",
+    "cy:run": "cypress run",
+    "lint": "eslint .",
+    "lint:fix": "eslint --fix .",
```

Comment:
I stole it from cypress-reaworld-app, what do you think?
    "lint": "eslint && prettier --check \"**/**.{ts,js,tsx}\" \"*.{json,md,yml}\"",

Reply:
Something was no yes with this command, I modified it a bit so now it should be fine